apps/server/src/modules/task/service/task.service.ts

```diff
@@ -41,4 +42,46 @@ export class TaskService {
 	async findById(taskId: EntityId): Promise<Task> {
 		return this.taskRepo.findById(taskId);
 	}
+
+	async deleteTasksByOnlyCreator(creatorId: EntityId): Promise<DomainOperation> {
+		const [tasksByOnlyCreatorId, counterOfTasksOnlyWithCreatorId] = await this.taskRepo.findByOnlyCreatorId(creatorId);
+
+		const promiseDeletedTasks = tasksByOnlyCreatorId.map((task: Task) => this.delete(task));
+
+		await Promise.all(promiseDeletedTasks);
```

Comment:
No check for the counter as in the other 2 methods?

Reply:
done

Reply:
done